api/environments/identities/serializers.py

```diff
@@ -28,44 +21,6 @@ class Meta:
         fields = ("id", "identifier", "identity_features", "environment")
 
 
-class EdgeIdentitySerializer(serializers.ModelSerializer):
-    identity_uuid = serializers.CharField(required=False)
-
-    class Meta:
-        model = Identity
-        fields = ("identifier", "environment", "identity_uuid")
```

Comment:
Looks environment was never used, nor it's needed.

Reply:
Did you verify this with Kyle?